tests/queries/0_stateless/01814_distributed_push_down_limit.sh

```diff
@@ -69,7 +69,7 @@ function test_distributed_push_down_limit_with_query_log()
         system flush logs;
         select read_rows from system.query_log
             where
-                event_date = today()
+                event_date >= today()
```

Comment:
Ok, and why today() remains here?

Reply:
My typo, fixed (and also in one more test).